app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Auth;
+use App\Models\User;
+class UserController extends Controller
+{
+    //
+    public function login(Request $req)
+    {
+        if(empty($req->user) && empty($req->password)){
+            $error = "Email or Password must be not empty!";
+            return View('login',['error'=>$error]);
```

Comment:
cách 1 dòng bên trên return
```suggestion
           
             return View('login',['error'=>$error]);
```

Reply:
completed!